python/runfiles/README.md

```diff
@@ -0,0 +1,64 @@
+# bazel-runfiles library
+
+This is a Bazel Runfiles lookup library for Bazel-built Python binaries and tests.
+
+Learn about runfiles: read [Runfiles guide](https://bazel.build/extending/rules#runfiles)
+or watch [Fabian's BazelCon talk](https://www.youtube.com/watch?v=5NbgUMH1OGo).
+
+## Typical Usage
+
+1. Depend on this runfiles library from your build rule, like you would other third-party libraries:
```

Comment:
I think making a not that projects can consume the PyPI parkage would be good. @rickeylev, do you remember the reasons why getting the runfiles lib from PyPI was the preferred approach?

Reply:
It's hard to say if one is strictly better than the other.

Pulling from pypi means the runfiles library version is decoupled from the rules version. This might be good -- you could use a newer version of runfiles to pick up a bug fix. Or it could be bad -- if we do something to couple the runfiles code to the rules logic (not sure what that would be, though). Both options could do more advanced things like generate pyc files, use c extensions, etc, so its just a question as to when those build steps happen (pre-built or on-demand).

That pip_parse needs to be setup seems like NBD. Most programs have _some_ external dependency, so you're probably going to have to do that no matter what.

Either way, I think we should document both. They're both public APIs we support, so might as well write them down.

Reply:
Updated the readme! Let me know what you think